packages/p2p/src/components/modal-manager/__mocks__/mock-modal-manager-context-provider.tsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react'
+import { useModalManagerContext } from '../modal-manager-context';
+import { Modal } from '@deriv/components';
+
+export function MockBuySellModal({ title, subtitle }: { title?: string; subtitle?: string }) {
+    const { is_modal_open, hideModal, showModal, useRegisterModalProps } = useModalManagerContext();
+
+    useRegisterModalProps({
+        key: 'MyAdsDeleteModal',
+        props: {
+            title: 'Title from BuySellModal',
+        } as unknown as Record<string, never>,
+    });
+
+    const showMyAdsDeleteModal = () => {
+        showModal({
+            key: 'MyAdsDeleteModal',
+            props: {},
+        });
+    };
+
+    return (
+        <Modal is_open={is_modal_open}>
+            {title && <div>BuySellModal with {title}</div>}
+            {title && subtitle && (
+                <div>
+                    BuySellModal with {title} and {subtitle}
+                </div>
+            )}
+            {!title && !subtitle && <div>BuySellModal</div>}
+            <button onClick={() => hideModal()}>Cancel</button>
+            <button onClick={showMyAdsDeleteModal}>Apply</button>
+        </Modal>
+    );
+}
+
+export function MockMyAdsDeleteModal({ title }: { title?: string }) {
+    const { is_modal_open, hideModal } = useModalManagerContext();
+    return (
+        <Modal is_open={is_modal_open}>
+            {title && <h1>{title}</h1>}
+            <h1>MyAdsDeleteModal</h1>
+            <button onClick={() => hideModal()}>Cancel</button>
```

Comment:
```suggestion
            <button onClick={hideModal}>Cancel</button>
```

Reply:
I couldn't set this to `{hideModal}` due to typing errors:
![Screenshot 2023-07-12 at 4 00 11 PM](https://github.com/farrah-deriv/deriv-app/assets/103016120/2ccc6d98-bd9e-4f65-9d8c-7782ebb76f54)